ship.config.js

```diff
@@ -25,14 +26,32 @@ module.exports = {
     exec('yarn eslint lerna.json --fix');
 
     // Update package dependencies
+    exec(
```

Comment:
This is getting out of hand and we need to find another solution.

We can go with:

- replacing the version values of `package.json` files and running `yarn` or
- using [Lerna `version` command](https://github.com/lerna/lerna/tree/master/commands/version) but it doesn't support our pinning strategy

Reply:
In the js client I'm about to fix this same problem with lerna version, what doesn't it support about  pinned dependencies? It seemed to work for me

Reply:
It seems to either use the exact strategy or not (`^`)—but we need more flexibility: exact pinning for dev dependencies and caret pinning for dependencies and peer dependencies. Ideally, Lerna shouldn't change our pinning start when upgrading, but it does.

Reply:
In ca7d9a73216f07ccb17a69ad95f9bcbd08001024 I now rely on Lerna to bump our deps to the exact strategy. In b8bcfbe47a8b51a79b8f19d85cb37da8855bde13 I group our local deps to `dependencies` instead of a mix between `devDependencies` and `peerDependencies`. It'll be easier to manager, doesn't impact the bundle size, and Lerna doesn't support bumping peer deps.